src/mcp_server_helm/commands/show.py

```
@@ -0,0 +1,100 @@
+import logging
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)